time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -170,8 +200,29 @@ def check_whether_current_user_owns_item(cls, data: dict):
 
     def get_all(self, conditions: dict = {}) -> list:
         conditions.update({"owner_id": self.current_user_id()})
+
+        if 'month' and 'year' in conditions:
+            month = int(conditions.get("month"))
+            year = int(conditions.get("year"))
+            conditions.pop('month')
+            conditions.pop('year')
+        elif 'month' in conditions:
+            month = int(conditions.get("month"))
+            year = get_current_year()
+            conditions.pop('month')
+        else:
+            month = get_current_month()
```

Comment:
The thing is that if they are not present in the filter you should actually skip that filter. It has better performance and better consistency: When a filter is not present it is because it is not needed. For less than this, we avoided having into account whether the deleted field was defined... and also `null`.

Reply:
This part may return an empty dictionary or whatever default value expects `find_all` for time entries for this parameter.